### ISSUE TITLE
Problem: It looks like the URLs for the docs redirect from index are not being rewritten correctly

### ISSUE THREAD

**Issue by ross-spencer** [artefactual/archivematica-docs#171]
**Version of the documentation:** 1.7

**Page (and section, if applicable) where the issue occurs:** https://www.archivematica.org/en/docs/archivematica-1.7/index/

**Description of the issue:**

Following links from he index e.g. import metadata: 

https://www.archivematica.org/en/docs/archivematica-1.7/index/user-manual/transfer/import-metadata/#import-metadata

Sees the URL written to include `/index/user-manual/..` when the correct URL should not include `/index`:

https://www.archivematica.org/en/docs/archivematica-1.7/user-manual/transfer/import-metadata/#import-metadata

**Comment by ross-spencer** [comment]
@sevein I wonder if this is related to some of the changes made for #137?

**Comment by fiver-watson** [comment]
This issue is not documentation related - it's related to something in the Django app we're using for our websites. I have found examples of working sites (including the Django docs themselves) that use the same sub-index structure in their docs. @sevein thinks he can fix it by tweaking our website search settings. 

I've filed the following internal ticket for AtoM - same fix could be applied for both websites: 

- https://projects.artefactual.com/issues/12455
